pmd-java/src/main/resources/category/java/errorprone.xml

```diff
@@ -1161,21 +1161,23 @@ should be annotated with @Test and @Ignore.
         <properties>
             <property name="xpath">
                 <value><![CDATA[
-//ClassOrInterfaceBodyDeclaration
-[../ClassOrInterfaceBodyDeclaration/Annotation/*/Name
+//MethodDeclaration
+    (: at least one other method with test annotation :)
+    [../../ClassOrInterfaceBody/MethodDeclaration/ModifierList/Annotation
         [pmd-java:typeIs('org.junit.Test')
```

Comment:
I think we should eventually convert this rule to a java rule and use TestRuleUtil to avoid the duplication of these annotations. I'll do it